src/main/java/com/ui/pages/LoginPage.java

```diff
@@ -43,6 +43,7 @@ public void verifyPageHeaders(){
         String title = homePageTitle.getText();
         Assert.assertEquals("Register", title, "Not navigated to the register page");
         utils.navigateToBack();
+        System.out.println("Git Test");
```

Comment:
can you print the title int eh priting statement

Reply:
Changes are made. Please review the code

Reply:
Looks good